### PR TITLE
9 - Upgrading to Polymer 2.x - part 2

### PR DESCRIPTION
**Pull request by dotpointer**
Upgrading to Polymer 2.x - part 2.

* Changed to ES6 classes, except behavior.
* Updating component and tests for linting and testing.
* Updating to ES6 syntax.

Component should work as before or better.

Issue is #9.

### CODE REVIEW

File: cosmoz-moment-timeago.html [hunk 1]
```diff
@@ -5,48 +5,62 @@
 </dom-module>
 
 <script type="text/javascript">
-	(function () {
+	/*global Cosmoz*/
+	const locale = 'en';
```
Comment (Wurper) [inline cosmoz-moment-timeago.html]: Maybe name it in all caps for a global constant? LOCALE

File: cosmoz-moment.html [hunk 2]
```diff
@@ -30,75 +30,82 @@
 
 		window.Cosmoz = window.Cosmoz || {};
 
-		var localSharedMoment = moment(),
-			locale = 'en',
+		const localSharedMoment = moment(),
 			momentElements = [];
```
Comment (Wurper) [inline cosmoz-moment.html]: Global constants? LOCAL_SHARED_MOMENT, MOMENT_ELEMENTS

**Review by Wurper — APPROVED** [review]

File: cosmoz-moment.html (hunk 2 above)
Comment (dotpointer) [reply cosmoz-moment.html]: Pushed a change.

File: cosmoz-moment-timeago.html (hunk 1 above)
Comment (dotpointer) [reply cosmoz-moment-timeago.html]: Pushed a change.

**Comment by dotpointer** [comment]
@Wurper Can you review again due to the constant changes please?

**Review by Wurper — APPROVED** [review]

**Pull request merged**